tests/Resources/ti.calendar.calendar.test.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2015-2017 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
+
+if (utilities.isAndroid() || utilities.isIOS()) {
```

Comment:
there's no real reason to narrow this to just ios/android

Reply:
We did this because our docs do not indicate that `Ti.Calendar` is supported on Windows...
http://docs.appcelerator.com/platform/latest/#!/api/Titanium.Calendar
http://docs.appcelerator.com/platform/latest/#!/api/Titanium.Calendar.Calendar